backend/capellacollab/projects/toolmodels/modelsources/git/crud.py

```diff
@@ -93,3 +93,11 @@ def update_git_model(
 
     db.commit()
     return db_model
+
+
+def delete_git_model(
+    db: Session,
+    git_model: DatabaseGitModel,
+) -> DatabaseGitModel:
+    db.delete(git_model)
```

Comment:
This fails if there is a linked backup job (similar to the scenario in #444). It should either fail with a good error message (listing the dependencies, which have to be removed first) or the button in the frontend should be disabled with a proper message.

Reply:
I've overlooked it, thanks for the clarification.